unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/Tests/LoadableShapesMiscTests.cs

```diff
@@ -52,14 +52,14 @@ public IEnumerator PreExistentShapeUpdate()
 
         // Set its shape as a BOX
         var componentId = TestHelpers.CreateAndSetShape(scene, entityId, CLASS_ID.BOX_SHAPE, "{}");
-        yield return scene.GetSharedComponent(componentId).routine;
+        yield return ((scene.GetSharedComponent(componentId)) as IDelayedComponent).routine;
 
```

Comment:
Maybe we should have a method for the component update instead of the routine it will be more elegant and more future error proof